catalog/rest/catalog-rest-endpoint/src/test/java/org/codice/ddf/endpoints/rest/TestRestEndpoint.java

```diff
@@ -618,8 +618,7 @@ public void testGetMetacardAsXml() throws Exception {
                 new Answer<BinaryContent>() {
```

Comment:
Would that mean I have to get rid of the method that it is a parameter in? Should line 617 be `when(framework.transform(isA(Metacard.class), anyString(), isNull(Map.class))).thenReturn(content);` ?